<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fileinfo</class>
 <widget class="QWidget" name="Fileinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>349</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Fileinfo</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>252</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>属性：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseFleBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>....</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>501</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="isDirCheckbox">
      <property name="text">
       <string>目录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isFileCheckaBox">
      <property name="text">
       <string>文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isSymLinkCheckBox">
      <property name="text">
       <string>符号链接</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isHiddenCheckBox">
      <property name="text">
       <string>隐藏</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isWritadleCheckBox">
      <property name="text">
       <string>读</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isWitableCheckBox">
      <property name="text">
       <string>写</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isExrcutableCheckBox">
      <property name="text">
       <string>执行</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="fileNameLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="fileSizeLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="fileCreatELinrRdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="fileModifuLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="fileReadLinrEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>107</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>文件名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>文件大小：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>创建时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>最后修改时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>最后访问时间：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
